src/components/common/research-pages.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  faBriefcase,
```

Comment:
I didn't actually realise these were all FA icons, we have these available as SVGs in `src/svg`. It's probably more efficient to use those but that difference might be negligible

Reply:
missed them there, have brought them in now